tests/foreman/cli/test_errata.py

```diff
@@ -547,12 +1390,34 @@ def test_positive_list_filter_by_org_name(self):
         1. erratum list --organization=<org name>
 
         @Assert: Errata is filtered by Org name.
-
-        @caseautomation: notautomated
-
         """
-
-    @stubbed()
+        org_erratum_list = Erratum.list({
+            'organization': self.org['name'],
+            'per-page': 1000
+        })
+        org_errata_ids = {
+            errata['errata-id']
+            for errata in org_erratum_list
+        }
+        errata_org_erratum_list = Erratum.list({
+            'organization': self.errata_org['name'],
+            'per-page': 1000
+        })
+        errata_org_errata_ids = {
+            errata['errata-id']
+            for errata in errata_org_erratum_list
+        }
+        self.assertEqual(len(org_errata_ids), self.org_product_erratum_count)
```

Comment:
It seem same code as the one starting on line 1370. Same method extraction could apply on this code, similar to the one I proposed.